client/my-sites/hosting/phpmyadmin-card/index.js

```diff
@@ -29,31 +29,41 @@ const trackOpenPhpmyadmin = () =>
 		bumpStat( 'hosting-config', 'open-phpmyadmin' )
 	);
 
-export default function PhpMyAdminCard( { disabled } ) {
-	const translate = useTranslate();
-	const dispatch = useDispatch();
-	const siteId = useSelector( getSelectedSiteId );
-
+export function useOpenPmaLink() {
 	const [ loading, setLoading ] = useState( false );
-	const [ isRestorePasswordDialogVisible, setIsRestorePasswordDialogVisible ] = useState( false );
+	const dispatch = useDispatch();
 
-	async function openPmaLink() {
-		setLoading( true );
-		try {
-			const { token } = await wpcom.req.post( {
-				path: `/sites/${ siteId }/hosting/pma/token`,
-				apiNamespace: 'wpcom/v2',
-			} );
+	const openPmaLink = useCallback(
```

Comment:
As you refactor the original component, what do you think about taking a chance to improve the naming convention? `openPmaLink` uses a cryptic acronym, and `openPHPmyAdmin` used in the command palette was much clearer. Or even `openPhpMyAdmin` that follows camelCaps more correctly.

Reply:
Sounds good, I will make the changes as suggested